E2E-tests/tests/committee/test_committee.py

```diff
@@ -35,6 +35,68 @@ def calculate_d_param_tolerance(pc_epochs_in_mc_epoch, d_param_p, d_param_t):
 
 class TestCommitteeDistribution:
 
+    @mark.committee_distribution
+    @mark.ariadne
+    @mark.xdist_group("governance_action")
+    def test_update_d_param(
+        self,
+        api: BlockchainApi,
+        config: ApiConfig,
+        current_mc_epoch,
+    ):
+        """
+        * get latest (n + 1) mc epoch ariadne parameters
+        * run update d param
+        * get next (n + 2) mc epoch ariadne parameters
+        * confirm that d param was updated
+        """
+        data = api.get_ariadne_parameters(current_mc_epoch + 1)
```

Comment:
I think we should use N+2 as current P/T. It may happen that we generate the same values as current and get an error.